src/docs/previews/context-menu/main/css.svelte

```diff
@@ -0,0 +1,276 @@
+<script lang="ts">
+	import { createContextMenu } from '$lib';
+	import { writable } from 'svelte/store';
+	import ChevronRight from '~icons/lucide/chevron-right';
+	import Check from '~icons/lucide/check';
+
+	const {
+		trigger,
+		menu,
+		item,
+		checkboxItem,
+		separator,
+		createSubMenu,
+		createMenuRadioGroup,
+	} = createContextMenu();
+
+	const { subMenu: subMenuA, subTrigger: subTriggerA } = createSubMenu();
+
+	const { radioGroup, radioItem, isChecked } = createMenuRadioGroup({
+		value: 'Hunter Johnston',
+	});
+
+	const personsArr = [
+		'Hunter Johnston',
+		'Thomas G. Lopes',
+		'Adrian Gonz',
+		'Franck Poingt',
+	];
+
+	const settingsSync = writable(true);
+	const hideMeltUI = writable(false);
+</script>
+
+<span class="trigger" melt={$trigger} aria-label="Update dimensions">
+	Right click me.
+</span>
+
+<div class="menu" melt={$menu}>
+	<div class="item" melt={$item}>About Melt UI</div>
+	<div class="item" melt={$item}>Check for Updates...</div>
+	<div class="separator" melt={$separator} />
+	<div
+		class="item"
+		{...$checkboxItem}
+		use:checkboxItem={{ checked: settingsSync }}
+	>
+		<div class="check">
+			{#if $settingsSync}
+				<Check class="icon" />
+			{/if}
+		</div>
+		Settings Sync is On
+	</div>
+	<div class="item" melt={$subTriggerA}>
+		Profiles
+		<div class="rightSlot">
+			<ChevronRight class="icon" />
+		</div>
+	</div>
+	<div class="menu subMenu" melt={$subMenuA}>
+		<div class="text">People</div>
+		<div melt={$radioGroup}>
+			{#each personsArr as person}
+				<div class="item" melt={$radioItem({ value: person })}>
+					<div class="check">
+						{#if $isChecked(person)}
+							<div class="dot" />
+						{/if}
+					</div>
+					{person}
+				</div>
+			{/each}
+		</div>
+	</div>
+	<div melt={$separator} class="separator" />
+
+	<div
+		class="item"
+		{...$checkboxItem}
+		use:checkboxItem={{ checked: hideMeltUI }}
+	>
+		<div class="check">
+			{#if $hideMeltUI}
+				<Check class="icon" />
+			{/if}
+		</div>
+		Hide Melt UI
+		<div class="rightSlot">⌘H</div>
+	</div>
+	<div class="item" melt={$item} aria-disabled="true">
+		Show All Components
+		<div class="rightSlot">⇧⌘N</div>
+	</div>
+	<div melt={$separator} class="separator" />
+	<div class="item" melt={$item}>
+		Quit Melt UI
+		<div class="rightSlot">⌘Q</div>
+	</div>
+</div>
+
+<style lang="postcss">
```

Comment:
Now that I think about it... We should not use lang="postcss" for these examples. Which I think shouldn't be a hassle to change

Reply:
There are more peculiarities with all of the CSS examples I would like to fix.

For example, I would like to remove all of the --tw variables from the examples. What do you think?

Reply:
I think that would be amazing tbh

Reply:
If that's alright, I would love to do all of those changes in a separate PR.

Reply:
Hi.

Mind if I weigh in...

I was already working on other css examples(the old way) and given this info, guess I should hit pause, right?

Also, @kaevupoiss for someone hoping to build on top those fixes, mind sharing some insight into how you plan to approach it. Was hoping I could help if that's an option.

Reply:
> If that's alright, I would love to do all of those changes to all of the existing CSS examples in a separate PR.

Awesome! Will merge this PR and leave it to you then 😄

Reply:
> Hi.
> 
> Mind if I weigh in...
> 
> I was already working on other css examples(the old way) and given this info, guess I should hit pause, right?
> 
> Also, @kaevupoiss for someone hoping to build on top those fixes, mind sharing some insight into how you plan to approach it. Was hoping I could help if that's an option.

Probably, yes! Feel free to coordinate over a GH discussion here, or in the Discord channel.

Reply:
@shyakadavis Feel free to write me in Discord. @revalbuilding

Reply:
@shyakadavis Feel free to write me in Discord. @revalbuilding